crates/viewer/re_space_view_spatial/src/ui.rs

```diff
@@ -439,54 +408,20 @@ pub fn picking(
             .tree
             .lookup_result_by_path(&instance_path.entity_path)
         else {
-            continue; // No data result for this entity, meaning it's no longer on screen.
+            // No data result for this entity means it's no longer on screen.
+            continue;
         };
 
         if !data_result.is_interactive(ctx) {
             continue;
         }
 
-        // Special hover ui for images and point clouds:
-        let is_depth_cloud = depth_images
-            .depth_cloud_entities
-            .contains(&instance_path.entity_path.hash());
-
-        let picked_image = if hit.hit_type == PickingHitType::TexturedRect || is_depth_cloud {
```

Comment:
Let's remember to test depth-cloud picking

Reply:
👍 did already (and ofc while working on it I broke it during iterations ;-)). But I'll make sure to give it another spin prior to merge